developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx

```diff
@@ -18,94 +18,103 @@ import ReactPlayer from 'react-player/lazy'
 
 ## <i class="fa-solid fa-square-chevron-right"></i>&nbsp;&nbsp;Options for running Weaviate
 
-You can run Weaviate as a managed service (Weaviate Cloud Services, or WCS), or manage your own instances using Docker, Kubernetes, or Embedded Weaviate.
+Weaviate Cloud Services (WCS) can run Weaviate for you as a managed service. You can also manage your own Weavaite instances using Docker, Kubernetes, or Embedded Weaviate.
 
-As mentioned before, the underlying Weaviate code is identical regardless of the method. There are some differences to be aware of.
+No matter how you run Weaviate, the underlying code is identical. However, there are some operational differences to be aware of.
 
 ### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;WCS (Recommended)
 
 [WCS, or Weaviate Cloud Services](https://console.weaviate.cloud/), is a managed SaaS service that requires no maintenance at your end.
 
-As it is managed by Weaviate (the company - the software is not sentient... *yet*), it is the fastest way to create a new instance of Weaviate and requires the least amount of effort for the users.
+As it is managed by Weaviate (the company - the software is not sentient... *yet*). WCS is the fastest way to create a new instance of Weaviate and requires the least amount of effort for users.
 
-Weaviate instances on WCS come pre-configured for convenience. They include a number of Weaviate "modules" by default, as well as built-in support for user authentication for security.
+The Weaviate instances in WCS are pre-configured for your convenience. They include a number of Weaviate modules by default. They also have built-in support for user authentication.
```

Comment:
I think `on` is preferable for the same reason as below, but I don't feel very strongly about it tbh.